google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1724,6 +1727,13 @@ public DefaultReadWriteTransactionOptionsBuilder setIsolationLevel(
           return this;
         }
 
+        public DefaultReadWriteTransactionOptionsBuilder setReadLockMode(
+            ReadLockMode readLockMode) {
+          transactionOptionsBuilder.setReadWrite(
+              ReadWrite.newBuilder().setReadLockMode(readLockMode));
```

Comment:
Here also: Can we update the existing ReadWrite object on transactionOptionsBuilder, instead of setting a new one? I foresee potential bugs in the future if we leave this as-is.